ci/build/test-wheels.sh

```diff
@@ -6,7 +6,7 @@ set -e
 # Show explicitly which commands are currently running.
 set -x
 
-ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
+ROOT_DIR=$(python -c "import os; print(os.path.dirname(os.path.realpath(r'${BASH_SOURCE:-$0}')));")
```

Comment:
Not sure about going with Python there. We can follow the symlinks with bash as well and that would introduce minimal changes here.

```suggestion
ROOT_DIR=$(cd "$(dirname "$(readlink -f "${BASH_SOURCE:-$0}")"  )"; pwd)
```

Reply:
On my side it is working as expected. But yes I've read that. In any case, I am not a core maintainer here so just suggesting things 😃

Reply:
@tupui No problem. Comments are always welcome and can help find a better way to support features.

I'm using a mac with 11.2.1 and it seems that 

```
readlink -f ./config.yaml
readlink: illegal option -- f
usage: readlink [-n] [file ...]
```

I feel if a newer version of mac can support this, maybe we can use this one. I don't like to introduce python as well and also it seems like windows can't be bootstrapped.  Let me workaround and also check the CI mac version.

Btw, really thank you for catching up on the issue introduced in this PR 😀